pegasus/cache/i18n/en-US.yml

```diff
@@ -810,3 +810,6 @@ en-US:
   processing_beyond_longdescription: "Processing is a programming language, development environment, and online community. Since 2001, Processing has promoted software literacy within the visual arts and visual literacy within technology. Initially created to serve as a software sketchbook and to teach computer programming fundamentals within a visual context, Processing evolved into a development tool for professionals. Today, there are tens of thousands of students, artists, designers, researchers, and hobbyists who use Processing for learning, prototyping, and production."
   alice_beyond_longdescription: "Using an innovative programming environment to support the creation of 3D animations, the Alice Project provides tools and materials for teaching and learning computational thinking, problem solving, and computer programming across a spectrum of ages and grade levels."
   sphero_beyond_longdescription: "Whether you're an educator or a parent, SPRK lessons give kids a fun crash course in programming robots while sharpening their skills in math and science."
+  compatibility_https_header: "Please enable HTTPS for Code.org."
```

Comment:
These strings are generated from a gsheet, so the changes should be made to the i18n gsheet instead of here.  We'll scoop up this file from staging after that is done.